### DISCUSSION TITLE
Separate buttons for highlighting and underlining text

### DISCUSSION THREAD

**Discussion by omarthegeek** [zim-desktop-wiki/zim-desktop-wiki#2462] [Ideas]
Hi. Currently the button showing an underlined text happens to highlight text in yellow. I figured I can edit the style.conf, and change it so that the button showing an underlined text actually underline text, but then later I needed to also highlight some text, and I don't want to have to pick between the two formatting styles.

I'm not sure if there are plugins to add extra buttons, but it could be a useful feature to add to the core software, where a separate highlight text button is added separate from the underlined text button.

**Comment by introt** [comment]
Extra buttons won't help without extra formatting with its separate syntax. This'll likely be addressed in tandem with markdown support (#26) - though it should be noted that the base spec iirc has neither underline nor highlight. See also #268 (includes workaround for more styles)
